### PR TITLE
Better dataset title

### PR DESCRIPTION
**Pull request by l-vincent-l**

### CODE REVIEW

File: apps/transport/lib/transport/dataset.ex [hunk 1]
```diff
@@ -207,21 +208,37 @@ defmodule Transport.Dataset do
   def filter_has_realtime, do: from d in __MODULE__, where: d.has_realtime == true
   def count_has_realtime, do: Repo.aggregate(filter_has_realtime(), :count, :id)
 
-  @spec get_by([{:slug, any}, ...]) :: Transport.Dataset.t | nil
-  def get_by(slug: slug) do
-    __MODULE__
-    |> where(slug: ^slug)
-    |> preload_without_validations()
+  @spec get_by(keyword) :: Dataset.t()
+  def get_by(options) do
```
Comment (antoine-de) [inline apps/transport/lib/transport/dataset.ex]: why don't you use argument destructuring there ?

File: apps/transport/lib/transport_web/templates/dataset/details.html.eex [hunk 2]
```diff
@@ -11,7 +11,10 @@
         <%= img_tag(@dataset.full_logo, alt: @dataset.title) %>
       </div>
       <div class="dataset__title">
-        <h1><%= @dataset.title%></h1>
+        <h1>
+          <%= Dataset.nom(@dataset) %> -
```
Comment (antoine-de) [inline apps/transport/lib/transport_web/templates/dataset/details.html.eex]: shouldn't we add a `Dataset` function to have this title ? this way we can reuse it

File: apps/transport/lib/transport/dataset.ex (hunk 1 above)
Comment (l-vincent-l) [reply apps/transport/lib/transport/dataset.ex]: It's a Keyword; not a map.
With keywork you can map just the first item of the list.

File: apps/transport/lib/transport_web/templates/dataset/details.html.eex (hunk 2 above)
Comment (l-vincent-l) [reply apps/transport/lib/transport_web/templates/dataset/details.html.eex]: Yes we can!

**Comment by Tristramg** [comment]
Repris dans #818

**Pull request closed without merging**